igvc2023/scripts/ramp_detect.py

```diff
@@ -53,22 +59,27 @@ def scanCallback(self, msg):
 
 		#----------main process----------
 		if node_flag == 3: # Only ramp detection. When detected, set node_flag to 4
-			local_goal_x, local_goal_y, _ = self.detectRamp(msg)
+			local_goal_x, local_goal_y, _, _ = self.detectRamp(msg)
 
 			if local_goal_x==-1: # can't detect ramp
 				return
 			elif np.linalg.norm([local_goal_x, local_goal_y]) < self.max_range: #<2.0
-				rospy.set_param("node_flag", 4)
+				node_index += 1
+				rospy.set_param("node_index", node_index)
 
 		elif node_flag == 4: # approach, climb, and down ramp. When finished, set node_flag to 1]
 			if self.process_id==0:
 				self.approachToRamp(msg)
 
 			elif self.process_id==1:
-				self.climbRamp(0.8)
+				self.climbRamp(self.climb_speed)
 
 			elif self.process_id==2:
-				self.downRamp(0.5) # in this function, set node_flag to 1
+				end_ramp = self.downRamp(self.down_speed) # in this function, set node_flag to 1
```

Comment:
誤解を生みそうなので、こちらのコメントは削除してしまった方が良いと思います。
```suggestion
				end_ramp = self.downRamp(self.down_speed)
```